src/pages/home/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from 'react';
+import { Navigate } from 'react-router-dom';
+import Navbar from '../../components/Navbar';
+import { getUserInfo } from '../../util/getUser';
+import { isUserLogged } from '../../util/loginUser';
+import './home.css';
+import UserPosts from './userPosts';
+interface ILogged {
+  isLogged: boolean;
+  userID: number;
+}
+
+const Home = () => {
+  const { isLogged, userID } = isUserLogged() as ILogged;

```

Comment:
You shouldn't have to do this type casting here. Something is probably wrong with the types in your function if you do

Reply:
Oh! i did not know it! i won't do it anymore!